Core.BackgroundJobs/Model/ReadModels/ScheduleUpdatesForDataProcessingRegistrationOverviewReadModelsWhichChangesActiveState.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Core.Abstractions.Types;
+using Core.DomainModel.BackgroundJobs;
+using Core.DomainServices.Repositories.BackgroundJobs;
+using Core.DomainServices.Repositories.GDPR;
+using Infrastructure.Services.DataAccess;
+
+namespace Core.BackgroundJobs.Model.ReadModels
+{
+    public class ScheduleUpdatesForDataProcessingRegistrationOverviewReadModelsWhichChangesActiveState : IAsyncBackgroundJob
```

Comment:
you have not scheduled this one to run...

Reply:
https://strongminds.atlassian.net/wiki/spaces/KITOS/pages/838303745/Creating+and+exposing+an+overview+read+model#Hangfire-jobs-to-re-trigger-rebuilds-based-on-evens-such-as-data-of-year-(contract-expiration-etc)